### DISCUSSION TITLE
Does parentData only work on layout files?

### DISCUSSION THREAD

**Discussion by StokleyO** [sveltejs/kit#10709] [Q&A]
I've been struggling for the past week to pass parentData to child routes when the data originates in a page.server.ts file. But when it originates in a layout.server.ts file, the parentData passes just fine. Is this how the parentData functionality was intended to be used or is there something I'm missing? Any help would be greatly appreciated

**Comment by teemingc** [comment]
Yes, this is intended. Only layout data gets passed down to child pages.
